timeseries/src/test/scala/com/criteo/cuttle/timeseries/TimeSeriesSpec.scala

```diff
@@ -28,4 +26,73 @@ class TimeSeriesSpec extends FunSuite {
           (job(1), TimeSeriesContext(ts(i), ts(i + 1)))
         })
   }
+
+  test("it should validate empty workflow") {
+    val workflow = Workflow.empty[TimeSeries]
+
+    assume(workflow.isInstanceOf[Workflow[TimeSeries]], "workflow is not timeseries workflow")
```

Comment:
Why are those type checking preconditions needed ?

Reply:
I am not sure what the issue is but maybe we should just add type annotation on Workflow.empty ?